### PR TITLE
New FCU

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
FCU in a trench, with heating and cooling valves, and two fans. @kevin-hereworks

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 1]
```diff
@@ -19,6 +19,19 @@
...
 
+FCU_RTM_DFSS2X_CHWZTC_SFVSC2X_HWZTC_STC_EFFC:
+  description: "FCU in a trench, with heating and cooling valves, and two fans."
+  is_canonical: true
+  implements:
+  - FCU
+  - RTM
+  - DFSS2X
+  - CHWZTC
+  - SFVSC2X
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:30]: Please be consistent with fan type applied - cannot mix DFSS and SFVSC

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -19,6 +19,19 @@
...
+  - HWZTC
+  - STC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:32]: How can the unit have supply temp control and also zone temp control?

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 3]
```diff
@@ -19,6 +19,19 @@
...
+  - EFFC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:33]: By definition, FCUs do not exhaust air - how can there be an exhaust fan on this unit?

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Apologies - this was only supply temp monitoring not control.

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 3 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Apologies - should have been under discharge fan, thank you. I have adjusted the new FCU entity type

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 4]
```diff
@@ -19,6 +19,17 @@
...
+FCU_RTM_DFVSC2X_CHWZTC_HWZTC_STM:
+  description: "FCU in a trench, with heating and cooling valves, and two fans."
+  is_canonical: true
+  implements:
+  - FCU
+  - RTM
+  - DFVSC2X
+  - CHWZTC
+  - HWZTC
+  - STM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:31]: This should be DTM for consistency (use supply or discharge exclusively) -- discharge fans, discharge temp, etc.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Comment by Ciaran-Oflannagain** [comment]
Thanks for your help @cstirdivant - I have updated to DTM as it is Discharge not Supply Air Monitoring. Thanks, Ciaran

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**